src/test/java/touro/peg/PlayMoveTest.java

```diff
@@ -3,12 +3,50 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.util.List;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 class PlayMoveTest {
-
+    public List<Move> legalMoves = List.of(
```

Comment:
Create a new class call LegalMovesFactory and have a method that returns the List.